pages/people.md

```diff
@@ -30,6 +30,7 @@ based on commits.
 | [Garvit Bansal](https://github.com/Garvit244)            | [Flipkart](https://www.flipkart.com)| [IST][ist] |
 | [Michael Wall](https://github.com/mjwall)                |                                     | [ET][et]   |
 | [Ravikrishna Adepu](https://github.com/adepuravikrishna) | [SAP Labs LLC](https://sap.com)     | [PDT][pdt] |
+| [Kenneth McFarland](https://github.com/kpm1985)          |                                     | [PST][pst] |
```

Comment:
Can you add the link for PST to bottom of markdown page?  Something like..

```
[pst]: https://www.timeanddate.com/time/zones/pst
```

Reply:
@mikewalch  I have inserted the PST link you have requested

Reply:
Not to be too pedantic, but there is a difference between PST and PDT. PDT is only used part of the year. PST is used all year round in regions without daylight savings, but only part of the year where daylight savings is used. The term "PT" is probably better for regions which switch back and forth throughout the year.

There is another contributor listed which incorrectly has "PDT" listed. It is probably better to fix that entry to be "PT" pointing to https://www.timeanddate.com/time/zones/pt , and reuse it for yours also (unless you really do mean that you're in PST all year round).

Reply:
Making them both PT seems like a good idea to me.  @kpm1985, do you want to make the change?  Otherwise, I can when I merge the change.

Reply:
Ill catch it right now to make it easier on you